cards/product-prominentvideo/template.hbs

```diff
@@ -52,10 +53,7 @@
 {{#if (any card.details (all card.CTA1 card.CTA1.url card.CTA1.label) (all card.CTA2 card.CTA2.url card.CTA2.label) card.feedbackEnabled)}}
```

Comment:
`card.feedbackEnabled` can be removed now that the feedback buttons aren't included in the `contentWrapper` div

Reply:
good catch! updated